tst/quick/Utils.tst

```diff
@@ -11,6 +11,9 @@ gap> M := GL(7, 5 ^ 2).1;;
 gap> N := ApplyFunctionToEntries(M, x -> x ^ 5);;
 gap> M = ApplyFunctionToEntries(N, x -> x ^ 5);
 true
+gap> M := IdentityMat(12, GF(7));;
+gap> Size(ReshapeMatrix(M, 48, 3)) = 48;
```

Comment:
Don't call `Size` on a matrix, use `NrRows`; or here, probably `DimensionsMat` would be best. Well... and/or just use a smaller example so we can actually preview it...

```suggestion
gap> M := IdentityMat(4);;
gap> ReshapeMatrix(M, 2, 8);
[ [ 1, 0, 0, 0, 0, 1, 0, 0 ], [ 0, 0, 1, 0, 0, 0, 0, 1 ] ]
gap> M := IdentityMat(12, GF(7));;
gap> DimensionsMat(ReshapeMatrix(M, 48, 3));
[ 48, 3 ]
```